### DISCUSSION TITLE
Documentation for plugins - additional information

### DISCUSSION THREAD

**Discussion by mp3monster** [fluent/fluent-bit#8158] [Ideas]
I would be be really good if the documentation for plugins  had some 'boiler plate information' at the top or the bottom of each plugin showing:
- Which OS' the plugin is available on  e.g  Windows: Yes  Linux: No  MacOS: only AArch64
- It would be good to know which release first saw the plugin introduced.

Some organizations can be a little cautious of just jumping to the latest version (even if it is the right thing to do).  Knowing platform(s) or deployment restrictions might exist can help when organizations are needing to consider multiple platforms.

**Comment by agup006** [comment]
+1 this idea is great
